jwst/pipeline/master_background.cfg

```diff
@@ -0,0 +1,5 @@
+name = "master_background" 
+class = "jwst.background.MasterBackgroundStep"
```

Comment:
Should the directory be master_background rather than just background?  (There already is a background subdirectory of jwst.)

Reply:
The code files in the PR are in jwst.master_background, so yes this should be updated.

Reply:
Yes! That is a bug.  I will write a unit test for this, as my current tests did not catch this bug.